openapi/smartevents_mgmt_v2.yaml

```diff
@@ -1808,6 +1808,7 @@ components:
       required:
       - kind
       - id
+      - name
```

Comment:
@r00ta This seemed to be missing from earlier (I wasn't expecting it to change in this PR!!)